lib/internal/test_runner/reporter/dot.js

```diff
@@ -23,6 +27,14 @@ module.exports = async function* dot(source) {
     }
   }
   yield '\n';
+  if (failedTests.length > 0) {
+    yield `\n${colors.red}Failed tests:${colors.white}\n\n`;
+      for (const test of failedTests) {
+        const indentation = indent(test.nesting);
```

Comment:
Does this behave correctly for nested tests

Reply:
why not?

Reply:
AFAIU, a nested test has test.nesting > 0. So it will be indented depending on the value. The way `spec-reporter` prints failing tests is without any additional indentation for nested tests. Depending on what is required, I can remove or keep it.